libs/sdk-ui-geo/package.json

```diff
@@ -122,6 +123,7 @@
         "stylelint-config-prettier": "^8.0.2",
         "stylelint": "^13.8.0",
         "ts-jest": "^26.3.0",
-        "typescript": "4.0.2"
+        "typescript": "4.0.2",
+        "@types/uuid": "^8.3.0"
```

Comment:
Please keep the items in alphabetical order (rush unfortunately does not do this by itself 😞 )

Reply:
Done https://github.com/gooddata/gooddata-ui-sdk/pull/1341/commits/58bb870aa7fd055f14a3a5f2031b5b8e205662ca